src/foam/u2/tag/Input.js

```diff
@@ -145,7 +145,8 @@ foam.CLASS({
     function fromProperty(p) {
       this.SUPER(p);
 
-      if ( ! this.hasOwnProperty('onKey') && p.validateObj ) this.onKey = true;
+      if ( ! this.hasOwnProperty('onKey') && (p.validateObj || p.onKey) ) this.onKey = true;
```

Comment:
What if they explicitly set p.onKey = false? Then it shouldn't be enabled if validatObj is true.

Reply:
@kgrgreer something like this should work, right?
```
if ( ! this.hasOwnProperty('onKey') this.onKey = p.onKey === undefined ? p.validateObj : p.onKey;
```

Reply:
if ( this.hasOwnProperty('onKey') ) this.onKey = this.onKey;
else if ( p.hasOwnProperty('onKey') ) this.onKey = p.onKey;
else this.onKey = p.valudateObj;